tests/test_remove_peaks_around_precursor_mz.py

```diff
@@ -0,0 +1,39 @@
+import numpy
+from matchms import Spectrum
+from matchms.filtering import remove_peaks_around_precursor_mz
+
+
+def test_remove_peaks_around_precursor_mz_no_params_or_precursor():
+    """No parameters provided and no precursor mz present."""
```

Comment:
Use defaults instead of no parameters.
```suggestion
def test_remove_peaks_around_precursor_mz_defaults_no_precursor():
     """Using defaults and no precursor mz present."""
```